Mage/src/main/java/mage/cards/ExpansionSet.java

```diff
@@ -171,7 +175,36 @@ public String toString() {
 
         return theBooster;
     }
+    
+    protected int PartnerCheck(List<Card> booster, boolean partnerAllowed, int max, int i){
```

Comment:
method should not start with a capital letter
what is int i ?

Reply:
i is used for checking whether a card can be replaced or safely added (for example consider the case where a partner is the first card, say Khorvath Brightflame, Card1, Card2... without the check the algorithm would replace the first card with its partner Sylvia Brightspear , Card1, Card2). The solution might be too hackish, so I can further work on readability. Would it be better to replace the two parameters with a single boolean?